code/modules/projectiles/updated_projectiles/gun_attachables.dm

```diff
@@ -371,6 +371,21 @@ Defined in conflicts.dm of the #defines folder.
 	accuracy_mod = CONFIG_GET(number/combat_define/min_hit_accuracy_mult)
 	scatter_mod = -CONFIG_GET(number/combat_define/low_scatter_value)
 
+/obj/item/attachable/mosinbarrel
+	name = "sniper barrel"
+	icon_state = "mosinbarrel"
+	desc = "A heavy barrel. CANNOT BE REMOVED."
+	slot = "under" //only way for it to work with a bayonet is to make this take the underbarrel slot. no more bipods.
```

Comment:
Why not just buff the mosin's baseline accuracy and get rid of this?

Reply:
this barrel is what allows it to stretch beyond the 32x32 pixel sprite border.

Second reason is this is how the SVD is set up as well. and I am trying to keep attachments consistent rather than snowflake.

Reply:
Fair and reasonable.